src/meta/osMapper.js

```diff
@@ -4,16 +4,37 @@ module.exports = Object.freeze({
             id: "Darwin",
             os: "MacOS",
             commands: { LINK: "open" },
+            installers: {
+                "node": `curl -fsSL https://webinstall.dev/node@stable | bash`,
+                "npm" : `npm install -g npm | bash`,
+                "ng" : `npm install -g @angular/cli | bash`,
+                "go": `curl -fsSL https://webinstall.dev/golang@1.14 | bash`,
+                "cargo": `curl https://sh.rustup.rs -sSf | sh`,
+            }
         },
         {
             id: "Linux",
             os: "Linux",
             commands: { LINK: "xdg-open" },
+            installers: {
+                "Node" : `curl -fsSL https://webinstall.dev/node@stable | bash`,
+                "npm" : `npm install -g npm | bash`,
+                "ng" : `npm install -g @angular/cli | bash`,
+                "go" : `curl -fsSL https://webinstall.dev/golang@1.14 | bash`,
```

Comment:
Will only install go 1.14. This is what I was talking about.

Reply:
Right now can be kept as static(being updated manually). Will be changed once a latest release link can be found or can be automated at further stage.
Second option could be making a script which builds the go and other things from the github repositories as that is updated regularly and link remains static